templates/partial/_footer.html

```diff
@@ -1,38 +1,50 @@
-<footer class="p-footer is-dark">
-    <div class="row">
-      <hr />
+<footer class="p-strip--suru-dark-grey">
```

Comment:
The bullets look a bit weird when the screen gets smaller.
<img width="541" alt="image" src="https://user-images.githubusercontent.com/57550290/164216895-d5f5d538-ac5d-47cb-a662-049f8cf713c5.png">

Reply:
I just checked, that's a bug in Vanilla - I'll file an issue!

Reply:
Issue filed: https://github.com/canonical-web-and-design/vanilla-framework/issues/4429